ui/shared/status/StatusChatInput.qml

```diff
@@ -500,7 +502,7 @@ Rectangle {
         anchors.leftMargin: 4
         anchors.bottom: parent.bottom
         anchors.bottomMargin: 16
-        visible: control.chatType === Constants.chatTypeOneToOne
+        visible: control.chatType === Constants.chatTypeOneToOne && !control.isStatusUpdateInput
```

Comment:
Aren't those mutually exclusive? Can a chat be one to one and be a status update?

Reply:
Yeap correct! Must have slept there. Unfortunately this has been merged already now.